activerecord/lib/active_record/associations.rb

```diff
@@ -1456,7 +1457,8 @@ def has_many(name, scope = nil, options = {}, &extension)
       #   Specifies type of the source association used by #has_one <tt>:through</tt> queries where the source
       #   association is a polymorphic #belongs_to.
       # [:validate]
-      #   If +false+, don't validate the associated object when saving the parent object. +false+ by default.
+      #   When set to +true+, validates new objects added to association when saving the parent object. +true+ by default.
+      #   If you want to ensure associated objects are revalidated on every update, use +validates_associated+.
```

Comment:
You're right, I got too enthusiastic in copy-and-pasting. It's `false` by default for `has_one` association, I'll revert this bad change.